### ISSUE TITLE
IMAT: PV Read error for count

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#1729]
PVs are not reading correctly from IMAT. On my local machine I run
```caget IN:IMAT:DAE:MON:5:C```
I get number back 1 in 3 times and error back the rest of the time
```
IN:IMAT:DAE:MON:5:C            0
CA.Client.Exception...............................................
    Warning: "Virtual circuit disconnect"
    Context: "op=0, channel=IN:IMAT:DAE:MON:5:C, type=DBR_TIME_LONG, count=1, ct
x="control-svcs-r3.isis.cclrc.ac.uk:5064""
    Source File: ..\getCopy.cpp line 92
    Current Time: Tue Oct 18 2016 13:58:36.433759323
..................................................................
```
Same for ```caget IN:IMAT:CS:SB:M3a_Counts``` montior seems to work without issue, blocks in the GUI seem to work without issue.

Found by Tino and reported to @GDH-ISIS will report to @FreddieAkeroyd for advice.
Tino has scripted a way round this in genie python but it is obviously not ideal.

**Comment by John-Holt-Tessella** [comment]
Running ```caget -a IN:IMAT:CS:SB:M3a_Counts``` I get
```
Read operation timed out: some PV data was not read.
IN:IMAT:CS:SB:M3a_Counts       <undefined> 0
CA.Client.Exception...............................................
    Warning: "Virtual circuit disconnect"
    Context: "op=0, channel=IN:IMAT:CS:SB:M3a_Counts, type=DBR_TIME_LONG, count=
1, ctx="control-svcs-r3.isis.cclrc.ac.uk:5064""
    Source File: ..\getCopy.cpp line 92
    Current Time: Tue Oct 18 2016 14:03:07.049425987
..................................................................
```

**Comment by John-Holt-Tessella** [comment]
Seem to be bad for all monitor bloacks tried M1a, M3b and M4a. But other block ok tried `X` and `J1_VGAP`.

**Comment by FreddieAkeroyd** [comment]
If I add    -w 10   to the CAGET it seems to be OK for me, the default timeout is one second so it suggests that sometimes things are running a bit slow. Monitors (which are GUI blocks) would be unaffected as they don't have a timeout as such. I think this particular issue is already fixed by #1681 - I try to return non-stale values, but sometimes refreshing the value takes longer than the default caget timeout; however blocks will always be "monitored" somewhere and hence fixed by #1681 but it would still be an issue if a non-monitored (non-block) DAE value  was read with too small a CAGET timeout
